lua/gitlab/actions/discussions/annotations.lua

```diff
@@ -133,3 +133,17 @@
 ---@field commit_id string  -- This will always be ""
 ---@field line_code string
 ---@field position NotePosition
+
+---@class RootNode: NuiTree.Node
```

Comment:
This annotation should actually be removed. I added it because it helped me get rid of some LSP warnings, but I didn't know the warnings were caused by a bad LSP setting. Now this annotation probably introduces more typing warnings with a properly set up LSP. I can fix this on Monday, or if you @harrisoncramer would like to merge the PR earlier, than please remove the annotation yourself.